bootcamp/contrib/student_15.py

```diff
@@ -0,0 +1,12 @@
+def CaseConversion(string):
```

Comment:
Function names should be lowercase, see [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names).